### PR TITLE
🏗 Add auto-merging group for ampproject dependencies

### PR DESCRIPTION
**Pull request by rcebulko**
Automerge won't work yet since `renovate` doesn't have write permissions, but this group of ampproject dependencies would be safe to auto-merge as long as Travis passes.

### CODE REVIEW

File: .renovaterc.json [hunk 1]
```diff
@@ -76,6 +76,14 @@
...
       "updateTypes": ["minor"],
       "groupName": "core dependencies (minor updates)"
+    },
+
+    {
+      "assignAutomerge": true,
+      "automerge": true,
+      "updateTypes": ["minor", "patch"],
+      "packagePatterns": ["^@ampprojects/"],
+      "groupName": "ampproject dependencies (minor & patch updates)"
```
Comment (rsimha) [inline .renovaterc.json:86]: We currently disable updates for everything under `dependencies`. Does that rule need to be modified for this group to take effect?

https://github.com/ampproject/amphtml/blob/3c4af2940d1a40e8541b2d643c4db6a22af6daaa/.renovaterc.json#L8-L10

File: .renovaterc.json [hunk 2]
```diff
@@ -76,6 +76,14 @@
       "paths": ["+(package.json)"],
       "updateTypes": ["minor"],
       "groupName": "core dependencies (minor updates)"
+    },
+
+    {
+      "assignAutomerge": true,
+      "automerge": true,
```
Comment (rsimha) [inline .renovaterc.json:83]: We currently use [`forking-renovate`](https://github.com/apps/forking-renovate), which only has read access to code:

image


Do you know if auto-merge can work at all with `forking-renovate`? Or does it need us to switch to the more powerful [`renovate`](https://github.com/apps/renovate)?

File: .renovaterc.json [hunk 3]
```diff
@@ -76,6 +76,14 @@
...
+      "updateTypes": ["minor", "patch"],
+      "packagePatterns": ["^@ampprojects/"],
```
[the remark this answers is not in the document]
Comment (rcebulko) [reply .renovaterc.json:85]: Nice catch :scream: fixed.

File: .renovaterc.json (hunk 1 above)
Comment (rcebulko) [reply .renovaterc.json:86]: You are indeed correct; fixed. Some of the subgroups may merit enabling dependencies, albeit in a follow-up PR/discussion.

File: .renovaterc.json (hunk 2 above)
Comment (rcebulko) [reply .renovaterc.json:83]: It looks like there's no clean way to make it actually "hit the merge button" in the way we'd like. There's some mention in the doc of bots merging PRs for renovate as well. I'll have to do some digging to figure out what approach is best for automerge, but in the meantime this grouping is still useful.

File: .renovaterc.json (hunk 1 above)
Comment (rsimha) [reply .renovaterc.json]: Absolutely! I think the runtime WG will be happy to see PRs directed to them for most of these dependencies:

https://github.com/ampproject/amphtml/blob/e1a66bce763efff45522ab77e15cdc7ff2aaebc7/package.json#L21-L29

Context: https://github.com/ampproject/amphtml/issues/24784#issuecomment-536339301

**Pull request merged**